be/src/exec/exec_node.cpp

```diff
@@ -552,7 +553,7 @@ Status ExecNode::create_vectorized_node(starrocks::RuntimeState* state, starrock
         *node = pool->add(new EsHttpScanNode(pool, tnode, descs));
```

Comment:
should be vectorized::EsHttpScanNode. you should rebase latest main branch.

Reply:
done